package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Fast, in-memory, typo-tolerant, full-text search engine written in TypeScript",
   "main": "./dist/cjs/lyra.js",
   "module": "./dist/esm/lyra.js",
+  "browser": "./dist/rn/lyra.js",
```

Comment:
I'd make the output directory name easier to read so that people could import it as:

```js
import { create } from '@lyrasearch/lyra/react-native'
```